Custom Data/FormFilling/App.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version = "1.0" encoding = "UTF-8" ?>
```

Comment:
Please bring the Custom Data sample inside the FormFilling folder

https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/tree/master/Form%20Filling

Reply:
Changed the folder structure.